cldrDict_sconscript

```diff
@@ -30,7 +30,7 @@ def createCLDRAnnotationsDict(sources, dest):
 	with codecs.open(dest, "w", "utf_8_sig", errors="replace") as dictFile:
 		dictFile.write(u"symbols:\r\n")
 		for pattern, description in cldrDict.iteritems():
-			dictFile.write(u"{pattern}\t{description}\tsome\r\n".format(
+			dictFile.write(u"{pattern}\t{description}\tnone\r\n".format(
```

Comment:
Please remove `None` here, as it is the implicit default. It will save us a lot of space.

```suggestion
			dictFile.write(u"{pattern}\t{description}\r\n".format(
```

Reply:
> Please remove `None` here, as it is the implicit default. It will save us a lot of space.

After some test, it looks like the default level is all when I remove all "none", emojis were not pronounced except if I was on "all" level.